react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import logo from './logo.svg';
+import './App.css';
+
+
+function CustomButton() {
```

Comment:
We want the CustomButton component to only define the "template" for the buttons. 
The background color and text props are going to be specified from the App component.